config/features/config_test.go

```diff
@@ -93,3 +94,23 @@ func TestValidateNetworkFlags(t *testing.T) {
 		})
 	}
 }
+
+func Test_parseBlacklistedRoots(t *testing.T) {
+	strings := []string{"0xf98e27558ac9ba27ab7d0d3b97d5742a4a68b5e3d7f33c520eda14c39df6368c",
+		"0x31604da144e1047b250ee87c5049774870e4a140e8eb0087f38ebc4584c2e7",
+		"0x4fcf04e7e4075962360a91be0c54c1d3f67237aa2ce07d83b0479971af3f7e78",
+	}
+	expected := [][32]byte{
+		{0xf9, 0x8e, 0x27, 0x55, 0x8a, 0xc9, 0xba, 0x27, 0xab, 0x7d, 0x0d, 0x3b, 0x97, 0xd5, 0x74, 0x2a, 0x4a, 0x68, 0xb5, 0xe3, 0xd7, 0xf3, 0x3c, 0x52, 0x0e, 0xda, 0x14, 0xc3, 0x9d, 0xf6, 0x36, 0x8c},
+		{0x4f, 0xcf, 0x04, 0xe7, 0xe4, 0x07, 0x59, 0x62, 0x36, 0x0a, 0x91, 0xbe, 0x0c, 0x54, 0xc1, 0xd3, 0xf6, 0x72, 0x37, 0xaa, 0x2c, 0xe0, 0x7d, 0x83, 0xb0, 0x47, 0x99, 0x71, 0xaf, 0x3f, 0x7e, 0x78},
+	}
+	hook := test.NewGlobal()
+	obtained := parseBlacklistedRoots(strings)
```

Comment:
I think you can get coverage over `BlacklistedBlock` with minimal effort if you change to the following:
```	defer Init(&Flags{})
	cfg := &Flags{
		BlacklistedRoots: parseBlacklistedRoots(strings),
	}
	Init(cfg)
```
	
Then, in the loop over expected:
```
		for _, root := range expected {
		require.Equal(t, true, BlacklistedBlock(root))
	}
```

Reply:
nice catch